internal/uvm/virtual_device.go

```diff
@@ -16,8 +16,13 @@ const (
 	VPCILocationPathIDType  = "vpci-location-path"
 	VPCIClassGUIDTypeLegacy = "class"
 	VPCIClassGUIDType       = "vpci-class-guid"
+	VPCIDeviceIDType        = "vpci"
```

Comment:
I thought we were moving to using more descriptive names for these (like `vpci-class-guid`, `vpci-location-path`)? What value do we actually pass for this device ID type?

Reply:
Yes, sorry, I need to add a `vpci-instance-id` option. This value is for device instance ID.

Reply:
Added.